src/pextlib1.0/readline.c

```diff
@@ -309,6 +315,19 @@ int RLHistoryCmd(ClientData clientData UNUSED, Tcl_Interp *interp, int objc, Tcl
 		}
 		s = Tcl_GetString(objv[2]);
 		write_history(s);
+	}  else if (0 == strcmp("append", action)) {
+		if (objc != 3) {
+			Tcl_WrongNumArgs(interp, 1, objv, "append filename");
+			return TCL_ERROR;
+		}
+		s = Tcl_GetString(objv[2]);
+		hist_file = fopen(s, "a");
```

Comment:
Thank you GitHub review for hiding my review comment against the version in one of the lower commits. Please add error handling here. See `mkdtemp.c` for good examples. E.g., use:

```tcl
if (NULL == (hist_file = fopen(s, "a"))) {
    Tcl_AppendResult(interp, "fopen(", s, "): ", strerror(errno), NULL);
    return TCL_ERROR;
}
```

Reply:
@neverpanic Thanks for the code, I went ahead and pasted it in verbatim :)

I also added some error-checking for the `fprintf` call; please let me know if it's okay. It's not as pretty as the code in `mktemp.c`, but it should do the trick.